core/geometry/src/polyface/Polyface.ts

```diff
@@ -558,27 +607,21 @@ export class IndexedPolyface extends Polyface { // more info can be found at geo
     return this.data.normalCount;
   }
   /** Test if `index` is a valid facet index. */
-  public isValidFacetIndex(index: number): boolean {
-    return index >= 0 && index < this.facetCount;
+  public isValidFacetIndex(facetIndex: number): boolean {
+    return facetIndex >= 0 && facetIndex < this.facetCount;
   }
   /** Return the number of edges in a particular facet. */
   public numEdgeInFacet(facetIndex: number): number {
     if (this.isValidFacetIndex(facetIndex))
       return this._facetStart[facetIndex + 1] - this._facetStart[facetIndex];
     return 0;
   }
-  /** ASSUME valid facet index. Return start index of facet in pointIndex arrays. */
-  public facetIndex0(index: number): number {
-    return this._facetStart[index];
-  }
-  /** ASSUME valid facet index. Return one past end index of facet in pointIndex arrays. */
-  public facetIndex1(index: number): number {
-    return this._facetStart[index + 1];
-  }
-  /** Create a visitor for this polyface */
-  public createVisitor(numWrap: number = 0): IndexedPolyfaceVisitor {
-    return IndexedPolyfaceVisitor.create(this, numWrap);
-  }
+  /** Given a valid facet index, return the index at which its face loop starts in the index arrays. */
+  public facetIndex0(facetIndex: number): number { return this._facetStart[facetIndex]; }
```

Comment:
For consistenct please move returns to a new line

Reply:
To be addressed in next commit.